test/steps/v2_steps.py

```diff
@@ -1881,111 +2076,37 @@ def algod_v2_client(context):
-    global_ints,
-    local_bytes,
-    local_ints,
-    app_args,
-    foreign_apps,
-    foreign_assets,
-    app_accounts,
-    extra_pages,
-):
-    application_id = 0
-    if operation == "none":
-        operation = None
-    else:
-        if (
-            hasattr(context, "current_application_id")
-            and context.current_application_id
-            and operation != "create"
-        ):
-            application_id = context.current_application_id
-        operation = operation_string_to_enum(operation)
-    dir_path = os.path.dirname(os.path.realpath(__file__))
-    dir_path = os.path.dirname(os.path.dirname(dir_path))
-    if approval_program == "none":
-        approval_program = None
-    elif approval_program:
-        with open(
-            dir_path + "/test/features/resources/" + approval_program, "rb"
-        ) as f:
-            approval_program = bytearray(f.read())
-    if clear_program == "none":
-        clear_program = None
-    elif clear_program:
-        with open(
-            dir_path + "/test/features/resources/" + clear_program, "rb"
-        ) as f:
-            clear_program = bytearray(f.read())
-    local_schema = transaction.StateSchema(
-        num_uints=int(local_ints), num_byte_slices=int(local_bytes)
-    )
-    global_schema = transaction.StateSchema(
-        num_uints=int(global_ints), num_byte_slices=int(global_bytes)
-    )
-    if app_args == "none":
-        app_args = None
-    elif app_args:
-        app_args = split_and_process_app_args(app_args)
-    if foreign_apps == "none":
-        foreign_apps = None
-    elif foreign_apps:
-        foreign_apps = [int(app) for app in foreign_apps.split(",")]
-    if foreign_assets == "none":
-        foreign_assets = None
-    elif foreign_assets:
-        foreign_assets = [int(asset) for asset in foreign_assets.split(",")]
-    if app_accounts == "none":
-        app_accounts = None
-    elif app_accounts:
-        app_accounts = [
-            account_pubkey for account_pubkey in app_accounts.split(",")
-        ]
+def create_transient_and_fund(context, transient_fund_amount):
+    context.transient_sk, context.transient_pk = account.generate_account()
+    fund_account_address(context, context.transient_pk, transient_fund_amount)
 
-    sp = context.app_acl.suggested_params()
-    context.app_transaction = transaction.ApplicationCallTxn(
-        sender=context.transient_pk,
-        sp=sp,
-        index=int(application_id),
-        on_complete=operation,
-        local_schema=local_schema,
-        global_schema=global_schema,
-        approval_program=approval_program,
-        clear_program=clear_program,
-        app_args=app_args,
-        accounts=app_accounts,
-        foreign_apps=foreign_apps,
-        foreign_assets=foreign_assets,
-        extra_pages=int(extra_pages),
-        note=None,
-        lease=None,
-        rekey_to=None,
+
+@given(
+    "I fund the current application's address with {fund_amount} microalgos."
```

Comment:
**NTS - add to sentence changes**

New sentence needed to test app account address functionality